scripts/feature_tests/inspection_test.sh

```diff
@@ -4,6 +4,11 @@ set -x
 
 METAL3_DIR="$(dirname "$(readlink -f "${0}")")/../.."
 
+# shellcheck disable=SC1091
+# shellcheck disable=SC1090
+# shellcheck disable=SC2046
+source "feature_test_vars.sh"
```

Comment:
This is still not working

Reply:
Since we have `METAL3_DIR` can we source it with respect to that in all the files

Reply:
Noted. Thanks

Reply:
Good catch.